Updates to Horizontal and Vertical Nav Pattern

sjcox-rh opened the pull request:
Added Horizontal nav masthead designs which includes addition of image
assets

sjcox-rh commented:
@LHinson Do you mind looking over this to see if I missed anything for both the vertical and horizontal masthead design.  Thanks!

dlabrecq commented on source/pattern-library/navigation/horizontal-navigation/index.md:
```diff
@@ -14,6 +14,16 @@ layout: page-tabs
     </ul>
     <p>Consider using <a href="{{site.baseurl}}pattern-library/navigation/vertical-navigation/">vertical navigation</a> if you need a solution that can easily scale for a large amount of menu items and easily adapt to small screen sizes. Horizontal menus can also be made responsive but it usually requires a transformation from horizontal to vertical. Since vertical menus are already in this forma ...
     <p>Jump to <a href="#example-overview-1">Standard Horizontal Navigation</a>, or <a href="#example-overview-2">Navbar Variations</a></p>
+    <h2 id="example-overview-2">Masthead Design</h2>
```
I downloaded and ran the changes, but the jump-to links appear to be broken. For example, the id "example-overview-2" appears twice in the page.

<h2 id="example-overview-2">Masthead Design</h2>
and
<h2 id="example-overview-2">Navbar Variations</h2>

dlabrecq commented on source/pattern-library/navigation/horizontal-navigation/index.md:
```diff
@@ -14,6 +14,16 @@ layout: page-tabs
+    <p>
+      <img src="{{site.baseurl}}assets/img/Masthead_HorizontalVariation.png" alt="Masthead Horizontal - Option 2" width="1400"/>
+    </p>
     <h2 id="example-overview-1">Standard Horizontal Navigation</h2>
```
The jump-to links are broken here as well. There are two "example-overview-1" IDs.

h2 id="example-overview-1">Masthead Design</h2>
and
h2 id="example-overview-1">With Primary Menu Icons</h2>

I believe "Masthead Design" should also be added to the jump-to links.

Please also squash the commits -- there are currently 4.

sjcox-rh replied:
@dlabrecq I can't seem to figure out how to squash the commits.  Also, is there a way to delete commits as well, 4 of the 6 commits were me testing since this is the first time I've done this.

Thanks for your help.

Will also make the updates you suggested.

sjcox-rh commented:
@dlabrecq I fixed the broken jump to links and also added jump to links for the masthead section for both vertical and horizontal.  I still haven't figured out how to squash commits, because now there are 8.  ugh, sorry about that.

dlabrecq commented:
The commits are getting a bit messy. You have a merge/revert with patternfly/master and another test commit/revert mixed in. We don't want all that in the history -- makes it difficult to back out changes if there is an issue.

If you're still using GitHub Desktop, please start using SourceTree instead. I can show you how to fix your commits and squash tomorrow?

If you don't have time for that, I created a new PR #263 with your changes. We can close this PR, but you'll need to review #263 to ensure I got everything. Whatever you prefer?

sjcox-rh commented:
@dlabrecq So sorry about that, I'm still learning and realized that it was a sloppy first try. Thanks for creating the new PR request.  I will go ahead and review that so you can go ahead and close this PR.  

However, would you mind if we met on Wednesday so you can show me how to clean up commits in the future?

dlabrecq commented:
Sure. If you can, save your existing workspace and I'll show you how I fixed it using SourceTree.

dlabrecq commented:
Closing in favor of #263.
